templates/framework/dashboard.html

```diff
@@ -113,6 +113,9 @@ <h1 class="page-header">My Posts</h1>
                     <div class="row col-md-10" style="margin: 0; padding: 0;">
                         {% include 'posts/all.html' %}
                     </div>
+		    <div class="row col-md-10" style="margin: 0; padding: 0;">
+                        {% include 'posts/github_stream.html' %}
```

Comment:
I believe that's github_feeds.html.
When I test it I can't find a github_stream.html in the posts templates.